main.py

```diff
@@ -39,11 +45,38 @@ def main(config):
         device=config.device,
         float_precision=config.float_precision,
     )
+    # The policy is used to model the probability of a forward/backward action
+    forward_config = OmegaConf.create(config.policy)
+    forward_config["config"] = config.policy.forward
+    del forward_config.forward
+    del forward_config.backward
+
+    backward_config = OmegaConf.create(config.policy)
+    backward_config["config"] = config.policy.backward
+    del backward_config.forward
+    del backward_config.backward
```

Comment:
Just a note for now. I am not super happy about this solution, since it is a bit convoluted and makes the code less readable. I have started working on figuring out a simpler and functional alternative way, but I don't have it yet. In any case, in practical terms: ok to merge to `rf_gfn` but for `main` I would like to find a simpler way.